src/main/java/org/terasology/ShatteredPlanes/FacetProviders/GaussFilter.java

```diff
@@ -60,45 +60,38 @@ public void setSeed(long seed) {
     @Override
     public void process(GeneratingRegion region) {
         ElevationFacet facet = region.getRegionFacet(ElevationFacet.class);
-        Rect2i worldRegionExtended = facet.getWorldRegion();
-        Rect2i worldRegion = worldRegionExtended.expand(-8, -8);
-
+        BlockAreac worldRegion = facet.getWorldArea().expand(-8, -8, new BlockArea(BlockArea.INVALID));
```

Comment:
I'd try to avoid `INVALID` if possible, although this boils down to the same result. I don't think saving on 6 write operations is worth it.

```suggestion
        BlockAreac worldRegion = new BlockArea(facet.getWorldArea()).expand(-8, -8);
```